static/app/actionCreators/projects.tsx

```diff
@@ -13,6 +14,13 @@ import LatestContextStore from 'sentry/stores/latestContextStore';
 import ProjectsStatsStore from 'sentry/stores/projectsStatsStore';
 import ProjectsStore from 'sentry/stores/projectsStore';
 import type {PlatformKey, Project, Team} from 'sentry/types';
+import {
+  type ApiQueryKey,
```

Comment:
Can we move this to a different import statement?

Reply:
Moved to its own line! For my own edification, is there a difference between importing separately or together?

Reply:
Separate import statements makes the transpilers and linters job a lot easier (and faster)